src/oop/object_independence/medium/q3/MainTest.java

```diff
@@ -0,0 +1,179 @@
+package oop.object_independence.medium.q3;
+
+import global.ObjectTest;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.tools.CustomAssertions._assertArrayEquals;
+import static global.tools.CustomAssertions._assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MainTest {
+    // Java
```

Comment:
```suggestion
    // Parsons
```

Reply:
This was originally Parsons, I commented here https://github.com/canvas-gamification/canvas-gamification-junit-tests/pull/673#discussion_r1213802720 that the formatting was incorrect because "question" isn't supposed to be attached behind, but you changed it Java. Is the change to Java unrelated?

Reply:
We decided for all medium questions to be partially given java